### PR TITLE
Meta: use new WHATWG boilerplate and license

### PR DESCRIPTION
**Pull request by domenic**
Do not merge until https://github.com/tabatkins/bikeshed/pull/1155 is merged.

***
<a href="https://whatpr.org/fullscreen/113.html" title="Last updated on Jan 5, 2018, 3:22 PM GMT (d90d0ce)">Preview</a> | <a href="https://whatpr.org/fullscreen/113/d6934f3...d90d0ce.html" title="Last updated on Jan 5, 2018, 3:22 PM GMT (d90d0ce)">Diff</a>

### CODE REVIEW

File: fullscreen.bs [hunk 1]
```diff
@@ -1,25 +1,12 @@
...
 Abstract: The Fullscreen API standard defines an API for elements to display themselves fullscreen.
-Logo: https://resources.whatwg.org/logo-fullscreen.svg
-Boilerplate: omit feedback-header, omit conformance, omit index, omit idl-index
-!Participate: <a href=https://github.com/whatwg/fullscreen>GitHub whatwg/fullscreen</a> (<a href=https://github.com/whatwg/fullscreen/issues/new>file an issue</a>, <a href=https://github.com/whatwg/fullscreen/issues>open issues</a>)
-!Participate: <a href=https://wiki.whatwg.org/wiki/IRC>IRC: #whatwg on Freenode</a>
-!Commits: <a href=https://github.com/whatwg/fullscreen/commits>GitHub whatwg/fullscreen/commits</a>
-!Commits: [SNAPSHOT-LINK]
-!Commits: <a href=https://twitter.com/fullscreenapi>@fullscreenapi</a>
-!Tests: <a href=https://github.com/w3c/web-platform-tests/tree/master/fullscreen>web-platform-tests fullscreen/</a> (<a href=https://github.com/w3c/web-platform-tests/labels/fullscreen>ongoing work</a>)
+Translation: ja https://triple-underscore.github.io/fullscreen-ja.html
```
Comment (annevk) [inline fullscreen.bs]: Hmm, should we do this in a separate commit and also acknowledge @triple-underscore? That's what I proposed for the Notifications API.

Comment (domenic) [reply fullscreen.bs]: Sure, I'll remove it from here.
